test/testcafe/spec/ChallengeFactorEmail_OTP_spec.js

```diff
@@ -0,0 +1,56 @@
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import ChallengeFactorPageObject from '../framework/page-objects/ChallengeFactorPageObject';
+import { ClientFunction, RequestMock } from 'testcafe';
+import factorRequiredEmail from '../../../playground/mocks/idp/idx/data/factor-verification-email';
+import success from '../../../playground/mocks/idp/idx/data/success';
+import invalidOTP from '../../../playground/mocks/idp/idx/data/error-email-verify';
+
+const validOTPmock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx')
+  .respond(factorRequiredEmail)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(factorRequiredEmail)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(success)
+
+const inValidOTPmock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx')
+  .respond(invalidOTP, 403)
+
+fixture(`Challenge Email Form`)
+
+async function setup(t) {
+  const identityPage = new IdentityPageObject(t);
+  await identityPage.navigateToPage();
+  await identityPage.fillIdentifierField('Challenge Email');
+  await identityPage.clickNextButton();
+  return new ChallengeFactorPageObject(t);
+}
+
+test
+  .requestHooks(validOTPmock)
+  (`challenge email factor with valid OTP form has right labels`, async t => {
+    const challengeFactorPageObject = await setup(t);
```

Comment:
One thing to note while writing these tests here is that for a long time I had the assertions for pageTitle within the same test as `challenge email factor with valid OTP`.  That added a bunch of await statements in the test which started causing flakeyness. Thats why i decided to have a test just to verify labels with few awaits.